src/txn/txn_rollback_to_stable.c

```diff
@@ -377,10 +377,14 @@ __rollback_ondisk_fixup_key(WT_SESSION_IMPL *session, WT_REF *ref, WT_PAGE *page
          * stop timestamp if the original update's commit timestamp is out of order. We may see
          * records newer than or equal to the onpage value if eviction runs concurrently with
          * checkpoint. In that case, don't verify the first record.
+         *
+         * If we have fixed the out-of-order timestamp, then the newer update which has been
+         * reinserted with the fixed timestamp may have a durable timestamp that is smaller than the
```

Comment:
If fixed timestamp is always an older timestamp then can we use the word "reinserted with the an older timestamp"